Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -586,7 +590,10 @@ dummyRideBooking = RideBookingRes
   estimatedDistance : Nothing,
   estimatedDuration : Nothing,
   rideScheduledTime : Nothing,
-  vehicleServiceTierType : Nothing
+  vehicleServiceTierType : Nothing,
+  tollConfidence : Nothing,
+  driversPreviousRideDropLocLat : Nothing,
```

Comment:
create an object for this : driversPreviousRIdeDropLoc : {
lat : Number, lon : Number}